src/oc/hocedit.cpp

```diff
@@ -142,8 +136,9 @@ void hoc_edit_quit(void)
 #endif
 }
 
+#ifndef OCSMALL
 extern "C" int emacs_exit(int status) {
-#if !OCSMALL
+
```

Comment:
@alexsavulescu : fails because of this I assume.

Reply:
Was failing because `WITHOUT_MEMACS` didn't end up in `nrnconf.h`